environments/demo/launch.sh

```diff
@@ -77,15 +70,9 @@ main() {
         exit 1
     fi
     set -e
-    init_dirs
-    pushd $WORKDIR > /dev/null
-        echo "Download docker compose files ..."
-        curl -L https://raw.githubusercontent.com/gravitee-io/gravitee-docker/master/environments/demo/common.yml -o "common.yml"
```

Comment:
the goal of this file is to download a local demo and run it. If you remove that, It will doesn't work anymore